docs/data/joy/customization/theme-global-variants/theme-global-variants.md

```diff
@@ -0,0 +1,125 @@
+# Global variant tokens
+
+<p class="description">Learn how to customize the global variant tokens.</p>
+
+## Prerequisite
+
+- Read the [global variant concept](/joy-ui/core-features/global-variant/) page.
+- Read the [theme tokens](/joy-ui/customization/theme-tokens/) page.
+- (optional) Completed [the tutorial](/joy-ui/getting-started/tutorial/).
+
+## How it works
+
+Joy has four built-in global variants, `plain` `outlined` `soft` and `solid`. The styles generated for each variant comes from the global variant tokens inside the theme palette.
+
+The tokens are composed of three things:
+
+- variant name
+- state
+- CSS property (`border`, `background` or `color`)
+
+For example, the `solidBg` refers to the **background** when the variant is **solid** at **initial** state.
+
+Another example, the `outlinedHoverBorder` refers to the **border color** when the variant is **outlined** at **hover** state.
+
+Since the values of these tokens are related to colors, they live inside the palettes so that they are configurable between color schemes:
+
+```js
+// theme
+{
+  colorSchemes: {
+    light: {
+      palette: {
+        primary: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+        neutral: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+        danger: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+        info: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+        success: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+        warning: {
+          plainColor: 'valid CSS color',
+          plainHoverBg: 'valid CSS color',
+          plainActiveBg: 'valid CSS color',
+          // ...other variant tokens
+        },
+      }
+    },
+    dark: {
+      // ...same structure with different values
+    }
+  }
+}
+```
+
+## Override default tokens
+
+We recommend to use button as a starting point to customize the global variants because it tends to have the most variants with interaction compare to other components.
```

Comment:
```suggestion
We recommend to use button as a starting point to customize the global variants because it tends to have the most variants with interaction compared to other components.
```